corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -342,13 +344,46 @@ def get_restore_class(user):
     return restore_class
 
 
+def _get_loadtest_factor(domain, user):
+    """
+    Gets the loadtest factor for a domain and user. Is always 1 unless
+    both the toggle is enabled for the domain, and the user has a non-zero,
+    non-null factor set.
+    """
+    if domain and ENABLE_LOADTEST_USERS.enabled(domain.name):
+        return getattr(user, 'loadtest_factor', 1) or 1
```

Comment:
ah you're right, i saw this line: https://github.com/dimagi/commcare-hq/pull/6400/files#diff-9d9075c8ac384b5859ac5aa7120651e5R234 and thought that it was protected from being less than 1 or Falsey, but that check is only on the form so potentially you could hard code the model to have a different